store/etcd.go

```diff
@@ -6,13 +6,22 @@ import (
 	"sync"
 	"time"
 
-	"github.com/coreos/go-etcd/etcd"
+	"golang.org/x/net/context"
+
+	"github.com/coreos/etcd/client"
 )
 
+func EnableEtcdDebug() {
```

Comment:
Just debugging. If the consumer wants to enable something like a 'Trace' debug level they can turn on etcd debug logging.